docs/meshcloud.policies.md

```diff
@@ -75,18 +75,19 @@ Your organization is fully free to define policies across the entire meshStack.
 
 ### How are meshPolicies evaluated
 
-As mentioned before, meshPolicies are built on top of meshStack's [tagging](./meshcloud.metadata-tags.md) system. This is done by evaluating two meshObjects on their tags. There is no "direction" in a meshPolicy: the meshObjects are evaluated bi-directionally. It does not matter if you have a meshPolicy for "meshCustomer & meshProject" or "meshProject & meshCustomer", the policy evaluation is equal.
-If a policy is defined, the two meshObjects' tags will be evaluated on a so-called set intersection. This means that for either tags, at least one value must be present on both sides for a successful evaluation. If neither side have the given tag, the policy is also successful.
+As mentioned before, meshPolicies are built on top of meshStack's [tagging](./meshcloud.metadata-tags.md) system. This is done by evaluating two meshObjects on their tags. There is the "direction" between the affected meshObject and authoritative meshObject in a meshPolicy: the meshObjects are evaluated unidirectionally.
+If a meshPolicy is defined, the two meshObjects' tags will be evaluated on a so-called set intersection. All affected meshObject tag values must be present within the authoritative meshObject tag values. Otherwise the meshPolicy is not compliant. If neither side have the given tag, the meshPolicy is also successful.
 To make this more clear, the table below describes certain conditions and their behavior when policies are applied on tags.
 
-In this example, we're looking at a meshPolicy between a meshProject's `environment` tag, and a meshCustomer's `environment` tag with the following allowed values: `dev`, `qa`, and `prod`.
+In this example, we're looking at a meshPolicy between a meshProject's `environment` tag, and a meshCustomer's `environment` tag with the following allowed values: `dev`, `qa`, and `prod`. The meshProject is the affected meshObject and the meshCustomer is the authoritative meshObject.
 
 | meshProject  | meshCustomer | Result | Explanation |
 | ------------ | ------------ | ------ | ----------- |
-| `prod`       | `prod`       | ✓      | Both tags contain the value `prod` |
-| `prod`       | `dev`, `qa`  | ✖      | There is no overlapping value |
+| `prod`       | `prod`       | ✓      | meshProject tag values are equal to meshCustomer tag values |
```

Comment:
How are meshPolicies evaluated section is the correct section because we want to show some examples how tag values are evaluated against each other. The What are some examples? section shows the possible combinations on meshObject level. This is independent of the tag evaluation.
And the user filtering will be added during the feature implementation. Currently this is out of scope.

Reply:
Sorry to open it up again :D

But 
e.g the explanation that "meshProject is not a subset of meshCustomer" is the outcome of the blocking violation seen here.
![image](https://user-images.githubusercontent.com/64793818/155507162-d38420df-c558-4817-a67d-9d2046ad8a02.png)
right? 

The evaluation process however is intersection logic which previously explained. Everything is considered _failed_ and not compliant. When it does not contain the value and count as not compliant. 
But the way treat a violation is different. Either logging it as violation via the periodical job of the PolicyViolationService or putting up the blocking logic, which we do with the evaluateOrThrowCall.

Thinking about it, another solution to make this clear is: name the section differently.
sth like: 
When is it a violation? for this section and the other
What impact does a violation have on on different meshObjects? (On the one with diagram. And then you can have table there and specify the impact more precisely, e.g disabling values and so on)
I think we do necessarily have to keep that document structure like it is or follow it that closely.
I can also make a proposal to the docs on another branch, what do you think?

Reply:
You can make a proposal. You can integrate your suggestion into this branch if you want. Then we don't need a separate branch.